### PR TITLE
Add errors_on to ModelCase

### PR DESCRIPTION
**Pull request by gjaldon**
refs: https://github.com/phoenixframework/phoenix/issues/853

### CODE REVIEW

File: installer/templates/ecto/model_case.ex [hunk 1]
```diff
@@ -17,6 +20,7 @@ defmodule <%= application_module %>.ModelCase do
       alias <%= application_module %>.Repo
       import Ecto.Model
       import Ecto.Query, only: [from: 2]
+      import unquote(__MODULE__)
```
Comment (josevalim) [inline installer/templates/ecto/model_case.ex]: I think this pattern is confusing for people stared. Can't we just repeat the name of the case? :)

**Comment by josevalim** [comment]
I have added one small comment, everything else is perfect :+1:

File: installer/templates/ecto/model_case.ex (hunk 1 above)
Comment (gjaldon) [inline installer/templates/ecto/model_case.ex]: Was debating that in my head for a bit. The module name would definitely be much clearer. :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**